addons/website_mass_mailing/static/src/js/website_mass_mailing.js

```diff
@@ -16,12 +17,26 @@ publicWidget.registry.subscribe = publicWidget.Widget.extend({
         'click .js_subscribe_btn': '_onSubscribeClick',
     },
 
+    /**
+     * @constructor
+     */
+    init: function () {
+        this._super(...arguments);
+        this._recaptcha = new ReCaptcha(this);
+    },
+    willStart: function () {
+        this._recaptcha.loadLibs();
```

Comment:
Not awaited ? :o

Reply:
Is it bad to let recaptcha do his thing and keep on initializing the widget without waiting recaptcha ? It will only be used on submit and to add the terms which it can do asynchronously.

Reply:
Not bad at all, but then you have to prevent the form to be send while recaptcha is not fully loaded right?

Reply:
Yes that's what I do while calling the `getToken` method